devops/Makefile

```diff
@@ -19,6 +19,7 @@ export BACKUP_MONGODB_APP_NAME := $(or $(BACKUP_MONGODB_APP_NAME), mongodb-backu
 export BACKUP_CONFIGMAP_NAME := $(or $(BACKUP_CONFIGMAP_NAME), backup-conf)
 export BYPASS_CRA_INCOME_VERIFICATION := $(or $(BYPASS_CRA_INCOME_VERIFICATION), false)
 export BYPASS_APPLICATION_SUBMIT_VALIDATIONS := $(or $(BYPASS_APPLICATION_SUBMIT_VALIDATIONS), false)
+export SWAGGER_ENABLED := $(or ${SWAGGER_ENABLED}, false)
```

Comment:
I do not see this variable used as a parameter anywhere below in the api-deploy task. Can you please check ?

![image](https://user-images.githubusercontent.com/62901416/157956886-411f3393-ee69-4b65-a719-4908bcb419e0.png)